SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift

```diff
@@ -68,7 +86,9 @@ extension HomeViewController: UICollectionViewDataSource, UICollectionViewDelega
         case self.testimonialsCollectionView:
             // Return Max pages = 4 and add 1 more for item "Ver más"
             return min(testimonialsData.count + 1, 5)
-        // Here can be added news cases
+        case self.lastestNewsCollectionView:
+            // Return Max pages = 4 and add 1 more for item "Ver más". Its another case becouse takes data from another Array
+            return min(lastestNewsData.count + 1, 5)
```

Comment:
repetitive code,  but in this case is short if you have time make it in a separate function